docker/README.md

```diff
@@ -194,6 +195,16 @@ For instance, to run a container with a registered Nuxeo instance:
 docker run -it -p 8080:8080 -e NUXEO_CLID=<NUXEO_CLID> nuxeo/slim:latest
 ```
 
+#### NUXEO_CONNECT_URL
+
+The value of `NUXEO_CONNECT_URL` is appended to `nuxeo.conf` at startup.
```

Comment:
maybe this is the option that is appended (key and value) and not just the value?

Reply:
What about:
> Configure the Connect URL with the `NUXEO_CONNECT_URL` value in `nuxeo.conf` at startup
?

Reply:
Like `NUXEO_PACKAGES` I would say:
> `NUXEO_CONNECT_URL` allows to configure the Connect URL. It is used for instance to fetch the Nuxeo packages .